ocpp/messages.py

```diff
@@ -360,43 +319,134 @@ class CallError:
 
             [<MessageTypeId>, "<UniqueId>", "<errorCode>", "<errorDescription>", {<errorDetails>}] # noqa
     """
+
     message_type_id = 4
 
-    def __init__(self, unique_id, error_code, error_description,
+    def __init__(self, unique_id,
+                 error_code,
+                 error_description,
                  error_details=None):
         self.unique_id = unique_id
         self.error_code = error_code
         self.error_description = error_description
         self.error_details = error_details
 
     def to_json(self):
-        return json.dumps([
-            self.message_type_id,
-            self.unique_id,
-            self.error_code,
-            self.error_description,
-            self.error_details,
-        ],
+        return json.dumps(
+            [
+                self.message_type_id,
+                self.unique_id,
+                self.error_code,
+                self.error_description,
+                self.error_details,
+            ],
             # By default json.dumps() adds a white space after every separator.
             # By setting the separator manually that can be avoided.
-            separators=(',', ':'),
-            cls=_DecimalEncoder
+            separators=(",", ":"),
+            cls=_DecimalEncoder,
         )
 
     def to_exception(self):
-        """ Return the exception that corresponds to the CallError. """
+        """Return the exception that corresponds to the CallError."""
         for error in OCPPError.__subclasses__():
             if error.code == self.error_code:
                 return error(
                     description=self.error_description,
                     details=self.error_details
                 )
 
-        raise UnknownCallErrorCodeError("Error code '%s' is not defined by the"
-                                        " OCPP specification", self.error_code)
+        raise UnknownCallErrorCodeError(
+            "Error code '%s' is not defined by the OCPP specification",
+            self.error_code,
+        )
 
     def __repr__(self):
-        return f"<CallError - unique_id={self.unique_id}, " \
-               f"error_code={self.error_code}, " \
-               f"error_description={self.error_description}, " \
-               f"error_details={self.error_details}>"
+        return (
+            f"<CallError - unique_id={self.unique_id}, "
+            f"error_code={self.error_code}, "
+            f"error_description={self.error_description}, "
+            f"error_details={self.error_details}>"
+        )
+
+
+def validate_payload(message: Union[Call, CallResult],
```

Comment:
Only out of curiosity: Why did you move the function here from its original position?

Reply:
To use the type hint for the message, which refers to the Call and CallResult class, the function has to be after the class definitions

Reply:
I see. You could use this import to postpone evaluation of the type hint/annotations, without moving code around:

```
from __future__ import annotations
```